test/integration/tracking/LegacyTrackerWPORG.spec.ts

```diff
@@ -63,11 +63,11 @@ describe( 'LegacyTrackerWPORG', function () {
 		Math.random = oldRandom;
 	} );
 
-	it( 'should always track in "devMode" and ignore tracking rate', () => {
+	it( 'should always track in "devMode" and ignore tracking rate when rate is > 0', () => {
```

Comment:
We probably need a better way to test this now because we can never be fully sure that random is above 0.1